### PR TITLE
Implement chrome.i18n.getMessage

### PR DESCRIPTION
**Pull request by kevinsawicki**
Initial implementation of [chrome.i18n.getMessage](https://developer.chrome.com/extensions/i18n)

- [x] Add tests

/cc @jlord 🍐 

Refs #5842

### CODE REVIEW

File: lib/renderer/chrome-api.js [hunk 1]
```diff
@@ -199,4 +199,6 @@ exports.injectTo = function (extensionId, isBackgroundPage, context) {
     setPopup () {},
     getPopup () {}
   }
+
+  chrome.i18n = require('./extensions/i18n.js')
```
Comment (kevinsawicki) [inline lib/renderer/chrome-api.js]: @zcbenz Are you okay breaking this out to a separate file so `chrome-api` doesn't grow too large?

File: lib/renderer/extensions/i18n.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+const {ipcRenderer} = require('electron')
+const fs = require('fs')
+const path = require('path')
+
+let metadata
+
+const getExtensionMetadata = () => {
+  if (!metadata) {
+    metadata = ipcRenderer.sendSync('CHROME_I18N_MANIFEST', chrome.runtime.id)
```
Comment (kevinsawicki) [inline lib/renderer/extensions/i18n.js]: @zcbenz Any tips for the best way to get the manifest of a devtool's extension from within the extension?

This approach works but not sure if you knew of something cleaner or pre-existing.

File: lib/renderer/chrome-api.js (hunk 1 above)
Comment (zcbenz) [inline lib/renderer/chrome-api.js]: 👍

File: lib/renderer/extensions/i18n.js (hunk 2 above)
Comment (zcbenz) [inline lib/renderer/extensions/i18n.js]: Current solution should be fine, I can't think of a better solution.

**Pull request merged**